protocol/communities_messenger_test.go

```diff
@@ -384,13 +386,13 @@ func (s *MessengerCommunitiesSuite) createCommunity() (*communities.Community, *
 	return createCommunity(&s.Suite, s.owner)
 }
 
-func (s *MessengerCommunitiesSuite) advertiseCommunityTo(community *communities.Community, user *Messenger) {
-	advertiseCommunityTo(&s.Suite, community, s.owner, user)
+func (s *MessengerCommunitiesSuite) advertiseCommunityTo(community *communities.Community, user *Messenger, admin *Messenger) {
```

Comment:
Can we keep using the word `owner` here? Since `admin` can mean the permission, which is under owner level.

Reply:
Also could you please invert the order? It is easier to reason about if `owner` is first (owner advertises community to user).